oauth-common/src/main/java/io/strimzi/kafka/oauth/validator/JWTSignatureValidator.java

```diff
@@ -113,6 +114,7 @@ public class JWTSignatureValidator implements TokenValidator {
      * @param readTimeoutSeconds The maximum time to wait for response from authorization server after connection has been established and request sent (in seconds)
      * @param enableMetrics The switch that enables metrics collection
      * @param failFast Should exception be thrown during initialisation if unable to retrieve JWKS keys
+     * @param ignoreKeyUse Should any key present in JWKS key set be considered a public key for signature checking
```

Comment:
Should we keep these in the right order in which they are specified in the method?

Reply:
Good catch. Strange that neither IDEA nor SpotBugs raised any warnings for this.